xml/System.Threading.Tasks/ValueTask`1.xml

```diff
@@ -377,9 +390,9 @@
         <ReturnType>TResult</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the result.</summary>
+        <value>The result.</value>
+        <remarks></remarks>
```

Comment:
Should it be noted here that `Result` will block if the `ValueTask` wraps a `Task` that's not completed and throw if it wraps a `Task` that is faulted or cancelled?

Also, unlike `Task<T>.Result`, `ValueTask<T>.Result` does not throw `AggregateException`, that might be worth noting too.